### PR TITLE
Kazoo 3577

### PR DESCRIPTION
**Pull request by jamesaimonetti**
Flushes XML config as well as caches on update/delete to trunkstore docs

### CODE REVIEW

**Comment by fenollp** [comment]
Build fails because get_balance_threshold/1 got removed here https://github.com/2600hz/kazoo/commit/226fbc525a2bab7775ad09c80dbe765223064ce6

**Comment by fenollp** [comment]
@jamesaimonetti welp the build didn't fail, it's xref that caught this! hehehe!

**Comment by fenollp** [comment]
LGTM

File: applications/teletype/src/templates/teletype_low_balance.erl [hunk 1]
```diff
@@ -76,7 +76,7 @@ handle_req(DataJObj) ->
     Macros = [{<<"system">>, teletype_util:system_params()}
               ,{<<"account">>, teletype_util:account_params(DataJObj)}
               ,{<<"current_balance">>, get_current_balance(DataJObj)}
-              ,{<<"threshold">>, teletype_util:get_balance_threshold(DataJObj)}
+              ,{<<"threshold">>, teletype_topup:get_balance(DataJObj)}
```
Comment (lazedo) [inline applications/teletype/src/templates/teletype_low_balance.erl]: teletype_topup:get_balance calls wht_util:current_balance which doesn't seem to be a threshold ?

**Pull request merged**